developer_guide/installation/system_requirements/setup_requirements_ubuntu_1404.rst

```diff
@@ -0,0 +1,117 @@
+Setup System Requirements on Ubuntu 14.04
+==========================================
+
+Here is a quick guide to setup the :doc:`system_requirements` requirements on Ubuntu 14.04.
+
+System installation
+-------------------
+
+Base dependencies
+*****************
+
+Before installing requirements, update repositories information. You'll have to install MySQL, Apache and PHP.
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo apt-get update
+    $ sudo apt-get install mysql-server
+    $ sudo apt-get install apache2
+    $ sudo apt-get install libapache2-mod-php5 php5-cli
+    $ sudo apt-get install php5-mysql php5-intl php5-curl php5-gd php5-mcrypt
+    $ sudo php5enmod mcrypt
+    $ sudo apt-get install php5-apcu
+
+.. note::
+    PHP 5.5 provided in Ubuntu 14.04 comes with the Zend OPcache opcode cache. Only the data cache provided by APCu is needed.
+
+Choosing the product storage
+****************************
+
+.. include:: /reference/technical_information/choose_database.rst.inc
+
+Based on this formula, either you need :ref:`installing-mongodb`, either you can directly go to the :ref:`system-configuration` section.
+
+.. _installing-mongodb:
+
+Installing MongoDB
+******************
+
+* Install MongoDB server
+
+.. code-block:: bash
+    :linenos:
+
+    $ sudo apt-key adv --keyserver keyserver.ubuntu.com --recv 7F0CEB10
+    $ sudo echo deb http://downloads-distro.mongodb.org/repo/debian-sysvinit dist 10gen | sudo tee /etc/apt/sources.list.d/mongodb-10gen.list > /dev/null
+    $ sudo apt-get update
+    $ sudo apt-get install mongodb-10gen=2.4.14
+
+.. note::
+
+    Akeneo PIM may work with versions 2.6 of MongoDB but not with 3.*.
```

Comment:
J'aurai mis l'information du 3.* en premier: `Akeneo PIM may not work with MongoDB 3.*. *the supported version is 2.6*`

ce sont des baby comments hein :)